src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,57 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+################################################################################
+#
+# Grassman Pluecker Ideal
+#
+################################################################################
+function grassman_pluecker_ideal(subspace_dimension::Int, ambient_dimension::Int)
+    return convert(Polymake.ideal.pluecker_ideal(subspace_dimension, ambient_dimension))
+end
+
+@doc Markdown.doc"""
+    grassman_pluecker_ideal([ring::MPolyRing], subspace_dimension::Int, ambient_dimension::Int)
+
+Given an ambient dimension and a subspace dimension return the ideal
+generated by the Plücker relations. If the ring is not specified return the ideal 
+in a multivariate polynomial ring over the rationals.
+
+# Examples
+```jldoctest
+julia> grassman_pluecker_ideal(2, 4)
+ideal(x[0]*x[5] - x[1]*x[4] + x[2]*x[3])
+
+julia> R, x = PolynomialRing(ResidueRing(ZZ, 7), "x" => (1:2, 1:3), ordering=:degrevlex)
+(Multivariate Polynomial Ring in 6 variables x[1, 1], x[2, 1], x[1, 2], x[2, 2], ..., x[2, 3] over Integers modulo 7, nmod_mpoly[x[1, 1] x[1, 2] x[1, 3]; x[2, 1] x[2, 2] x[2, 3]])
+
+julia> grassman_pluecker_ideal(R, 2, 4)
+ideal(x[1, 2]*x[2, 2] + 6*x[2, 1]*x[1, 3] + x[1, 1]*x[2, 3])
+```
+
+The Grassman Plücker Ideal is the homogenous ideal generated by the relations defined by 
```

Comment:
Minor nitpick. Also, I think @wdecker wanted to have a look and comment, but I think he can also do this if this PR is merged and we adjust later, should he have any remarks. *shrug*
```suggestion
The Grassman-Plücker ideal is the homogenous ideal generated by the relations defined by 
```